cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,67 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] (PATH | RESOURCE_ALIAS RESOURCE_ID)",
 	Short:     "Send a DELETE request",
 	Long:      "Send a DELETE request to the given path.",
 	RunE:      run,
 	ValidArgs: urls.Resources(),
 }
 
+var usageTemplate = `Usage:{{if .Runnable}}
```

Comment:
If we will adapt this template for other command we should move it to some utils so we don't need to move it when doing refactor of the other commands.

Reply:
this template was picked up from cobra's internal help/usage template function: https://github.com/spf13/cobra/blob/bcfcff729ecdeb4072ef6f2a9c0b5e4ca1853206/command.go#L546